solver/llbsolver/solver.go

```diff
@@ -269,13 +269,17 @@ func notifyCompleted(ctx context.Context, v *client.Vertex, err error, cached bo
 	pw.Write(v.Digest.String(), *v)
 }
 
-var AllowNetworkHostUnstable = false // TODO: enable in constructor
+var AllowNetworkHostUnstable = false       // TODO: enable in constructor
+var AllowSecurityUnconfinedUnstable = true // TODO: enable in constructor
 
 func supportedEntitlements() []entitlements.Entitlement {
 	out := []entitlements.Entitlement{} // nil means no filter
 	if AllowNetworkHostUnstable {
 		out = append(out, entitlements.EntitlementNetworkHost)
 	}
+	if AllowSecurityUnconfinedUnstable {
+		out = append(out, entitlements.EntitlementSecurityUnconfined)
```

Comment:
Where is this validated? I think it is missing the condition in `ValidateEntitlements()`